packages/pyright-internal/src/localization/localize.ts

```diff
@@ -158,6 +158,8 @@ async function loadStringsFromJsonFile(locale: string): Promise<StringLookupMap
             return (await import(/* webpackChunkName: "ja" */ './simplified.nls.ja.json')).default;
         case 'ko':
             return (await import(/* webpackChunkName: "ko" */ './simplified.nls.ko.json')).default;
+        case 'pl':
+            return (await import(/* webpackChunkName: "nl" */ './simplified.nls.pl.json')).default;
```

Comment:
```suggestion
            return (await import(/* webpackChunkName: "pl" */ './simplified.nls.pl.json')).default;
```

Reply:
Well that would have been unfortunate! I was completely convinced by seeing the language change from English...

Reply:
Oh, I see it likely makes no difference...

Reply:
https://github.com/microbit-foundation/pyright/pull/36/commits/1fc5f9b30c024fa08f2dbfa2effe59b3eb47e141
and updated in the editor PR too

Reply:
Looks like they just get bundled together?